src/components/breadcrumb/breadcrumb.html

```diff
@@ -1,12 +1,12 @@
 <div class="bx--breadcrumb">
   <div class="bx--breadcrumb-item">
-    <a href="#" class="bx--link">Breadcrumb 1</a>
+    <a href="#" class="bx--link" tabindex="0">Breadcrumb 1</a>
```

Comment:
Did you try this without `tabindex="0"`? Usually for links, you don't need to add this since it should be tabbable by default.